### PR TITLE
Writer api

### PR DESCRIPTION
**Pull request by piyusql**

### CODE REVIEW

File: bridgeql/django/urls.py [hunk 1]
```diff
@@ -7,14 +7,18 @@
...
-from bridgeql.django.bridge import read_django_model
+from bridgeql.django import bridge
 from bridgeql.django.settings import bridgeql_settings
 from bridgeql.django.views import index, generate_bridgeql_schema
 
 bridgeql_settings.validate()
 
 urlpatterns = [
-    path('reader/', read_django_model, name='bridgeql_django_read'),
+    path('reader/', bridge.read_django_model, name='bridgeql_django_read'),
```
Comment (piyusql) [inline bridgeql/django/urls.py:17]: Let's have standard practice

CRUD - create/read/update/delete
create - POST
read - GET

- add a test case for big enough params to get raise "HTTP 413 Content Too Large" exception and return the same
- thinking, if there will be any such case to get these type of real requests.
- what will be the solution in the case, should we allow to use POST ?

update - PATCH/PUT (do not create new record)
delete - DELETE

**Review by piyusql — COMMENTED** [review]
Overall looks good, few things to address before merge.

File: bridgeql/django/settings.py [hunk 2]
```diff
@@ -11,7 +11,10 @@
 
 DEFAULTS = {
     'BRIDGEQL_RESTRICTED_MODELS': {},
-    'BRIDGEQL_AUTHENTICATION_DECORATOR': '',
+    'BRIDGEQL_AUTHENTICATION_DECORATOR': {
```
[the remark this answers is not in the document]
Comment (preyunk) [reply bridgeql/django/settings.py:14]: lets add a feature request in issues section

File: bridgeql/django/bridge.py [hunk 3]
```diff
@@ -33,20 +34,29 @@ def read_django_model(request):
...
-        else:
+            msg = 'Added new %s model, pk = %s' % (
+                obj._meta.model.__name__,
+                obj.pk
+            )
+        elif mo.instance and request.method == 'PATCH':
             obj = mo.update(params)
-        res = {'data': obj.id, 'message': 'Updated fields %s' % (
-            ", ".join(params.keys())), 'success': True}
+            msg = 'Updated fields %s' % ", ".join(params.keys())
```
[the remark this answers is not in the document]
Comment (preyunk) [reply bridgeql/django/bridge.py]: Lets add it as a feature request.

File: bridgeql/django/models.py [hunk 4]
```diff
@@ -138,6 +146,58 @@ def validate_fields(self, query_fields):
         return True
 
 
+class ModelObject(object):
+    def __init__(self, app_label, model_name, **kwargs):
+        self.model_config = ModelConfig(app_label, model_name)
+        self.instance = None
+        self.db_name = kwargs.pop('db_name', None)
```
[the remark this answers is not in the document]
Comment (preyunk) [reply bridgeql/django/models.py]: updated to `bridgeql_writer_db`

**Review by preyunk — APPROVED** [review]

**Pull request merged**